kubernetes/linux/main.sh

```diff
@@ -74,6 +121,15 @@ setCloudSpecificApplicationInsightsConfig() {
             echo "export APPLICATIONINSIGHTS_ENDPOINT=$APPLICATIONINSIGHTS_ENDPOINT" >>~/.bashrc
             source ~/.bashrc
             ;;
+         "bleu")
+            # Bleu cloud is a new cloud environment, we dont have AI in this cloud yet so using Public cloud for now.
+            # update this once we have AI in Bleu cloud.
+            APPLICATIONINSIGHTS_AUTH="NzAwZGM5OGYtYTdhZC00NThkLWI5NWMtMjA3ZjM3NmM3YmRi"
```

Comment:
please update our doc if applicable

Reply:
> please update our doc if applicable

This should be addressed with Bleu specific AI resources as mentioned in the comments. we are going with Public preview AI resource until we have AI resources in Bleu cloud.